src/components/Autocomplete/components/ComboBox/tests/ComboBox.test.tsx

```diff
@@ -499,6 +499,89 @@ describe('<ComboBox/>', () => {
       };
     });
   });
+
+  describe('empty state', () => {
+    const EmptyState = () => <div>No results</div>;
+
+    it('renders an empty state when no options are passed in', () => {
+      const comboBox = mountWithAppProvider(
+        <ComboBox
+          options={[]}
+          selected={[]}
+          textField={renderTextField()}
+          onSelect={noop}
+          emptyState={<EmptyState />}
+        />,
+      );
+
+      comboBox.simulate('click');
+      expect(comboBox.find(EmptyState).exists()).toBe(true);
```

Comment:
@solonaarmstrong probably has more opinions on this. Is our general pattern
```tsx
expect(comboBox.find(EmptyState).exists()).toBe(true);
```

or

```tsx
expect(comboBox.find(EmptyState)).toHaveLength(1);
```

Reply:
We favour the latter. https://github.com/Shopify/web-foundation/blob/master/Best%20practices/Enzyme.md#expectations

Reply:
We are doing `expect(comboBox).toContainComponent(EmptyState)` in `web`. I'll update to use `expect(comboBox.find(EmptyState)).toHaveLength(1);` though if we don't have the new matchers in this project.

Reply:
Also, the entire `ComboBox.test.tsx` is using 

`expect(comboBox.find(EmptyState).exists()).toBe(true);`